CalendarApp/BookingRoom/Models/GoogleConnection/CalendarConnection.cs

```diff
@@ -0,0 +1,47 @@
+﻿using BookingRoom.Models.GoogleConnection;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Services;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Web;
+
+namespace BookingRoom.Models
+{
+    /// <summary>
+    /// Class for creating Google Caledar Service
+    /// </summary>
+    public class CalendarConnection : ICalendarConnection
+    {
+        private X509Certificate2 certificate;
+        private ServiceAccountCredential credential;
+        private BaseClientService.Initializer initializer;
+        private CalendarService service;
+        /// <summary>
+        /// Constructor for connecting to Gmail Account
+        /// </summary>
+        public CalendarConnection(string serviceEmail,string password,string applicationName)
```

Comment:
I know I'm nitpicking but is it possible to add spaces after ','s when you are listing parameters.
The code should be able to breathe )